HMCL/src/main/resources/assets/lang/I18N_zh_CN.properties

```
@@ -955,6 +955,30 @@ repositories.chooser.title=选择 JavaFX 下载源
 resourcepack=资源包
 resourcepack.download.title=资源包下载 - %1s
 
+reveal.in_explorer=在资源管理器中查看
+
+schematics=原理图
+schematics.add=添加原理图
+schematics.add.failed=添加原理图失败
+schematics.back_to=返回到 “%s”
```

有些字体会将引号以全角显示，这时候加个空格会导致 “返回到” 和 “%s” 之间间隔过长。

```suggestion
schematics.back_to=返回到“%s”
```